src/main/java/com/onelogin/sdk/model/Role.java

```diff
@@ -4,24 +4,42 @@
 
 public class Role {
 
-	protected long id;
-	protected String name;
-
-	public Role(long id, String name) {
-		this.id = id;
-		this.name = name;
-	}
-
-	public Role(JSONObject data) {
-		id = data.optLong("id");
-		name = data.optString("name", null);
-	}
-
-	public long getID() {
-		return id;
-	}
-
-	public String getName()	{
-		return name;
-	}
+    protected long id;
+    protected String name;
+
+    public Role(long id, String name) {
+        this.id = id;
+        this.name = name;
+    }
+
+    public Role(JSONObject data) {
+        id = data.optLong("id");
+        name = data.optString("name", null);
+    }
+
+    public long getID() {
+        return id;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    @Override
+    public int hashCode() {
+        final int prime = 31;
```

Comment:
What is that 31?

Reply:
It's the auto generated hashCode function implementation provided by Eclipse. Should I implement it in any other way ?